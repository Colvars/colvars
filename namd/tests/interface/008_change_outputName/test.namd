# -*- tcl -*-


source ../Common/common.namd
source ../Common/measure_net_force_torque.tcl

stepspercycle 5

colvars                 on
colvarsConfig           test.in

if { [file exists test.prerun.tcl] > 0 } {
    source test.prerun.tcl
}

run 10

outputName test.alt

run 10

source ../Common/delete_tmp_files.tcl
