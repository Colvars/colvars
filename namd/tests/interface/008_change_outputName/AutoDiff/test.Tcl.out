TCL: Running for 10 steps
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: Setting parameter outputName to test.alt
TCL: Running for 10 steps
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: NUM. ATOMS REQUESTED:  0
TCL: NUM. ATOMS USED:       0
TCL: NUM. ATOM GROUPS USED: 2
TCL: TOTAL FORCE:  { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
TCL: TOTAL TORQUE: { 0.0000000000000000 0.0000000000000000 0.0000000000000000 }
